ocaml/xapi/extauth_plugin_ADwinbind.ml

```diff
@@ -345,6 +400,25 @@ module Wbinfo = struct
     | _ ->
         Error (generic_ex "Invalid domain user name %s" uname)
 
+  let domain_and_user_of_uname uname =
```

Comment:
The `of_uname` is redundant. A function always depends in its parameters. There are similar names below. I'm not sure this convention is too useful - but it also does not hurt. If you prefer long names, `split_into_domain_and_user` would work, or a shorter `split` as well.

Reply:
yes,  it sounds redundant, but I prefer this name from caller side, as we know what is outputted from the function.